File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.cpp
class Solution {
public:
    vector<bool> kidsWithCandies(vector<int>& candies, int extraCandies) {
        int m=-1;
        int n=candies.size();
        vector<bool> result;
        for(int i=0;i<n;i++)
        {
            m=max(candies[i],m);
        }
        for(int i=0;i<n;i++)
        {
            if(candies[i]+extraCandies>=m)
            {
                result.push_back(true);
            }
            else
            {
                result.push_back(false);
            }
        }
        return result;
    }
};

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        answer=[]
        maxCandies=max(candies)
        for i in range(len(candies)):
            if candies[i]+extraCandies < maxCandies:
                answer.append(False)
            else:
                answer.append(True)
        return answer